target_chains/cosmwasm/deploy-scripts/package.json

```diff
@@ -28,6 +28,7 @@
   "devDependencies": {
     "@types/adm-zip": "^0.5.0",
     "@types/yargs": "^17.0.18",
-    "prettier": "catalog:"
+    "prettier": "catalog:",
+    "ts-node": "^10.9.2"
```

Comment:
We should probably move `ts-node` to the catalog too since we should be managing it alongside typescript, and ideally we should upgrade the tooling across the whole monorepo simultaneously.